### PR TITLE
update #1 in README.md

### PR DESCRIPTION
**Pull request by herohua**

### CODE REVIEW

**Comment by herohua** [comment]
comment added via GitHub API

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # CodeFix
 Code fix project based on Roslyn
+
+Update #1
```
Comment (herohua) [inline README.md]: good

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
 # CodeFix
```
Comment (herohua) [inline README.md]: Comment via GitHub API

**Comment by Huachao** [comment]
no

**Comment by herohua** [comment]
reopen

**Pull request merged**